tests/ci/check_so_interface.py

```diff
@@ -104,49 +130,50 @@
     for node in ast.walk(frozen):
         if isinstance(node, ast.FunctionDef) and node.name in spec_args.keys():
             freeze.update({node.name: node})
+print("FROZEN")
```

Comment:
There's a lot of these print checks I put along the way to debug, and visualize the flow of my code. I haven't deleted them yet because I wanted to keep them in here to remember good places/dictionaries and lists to check in my code for future debugging. A final version of this code will definitely have these removed.